nbconvert/exporters/exporter.py

```diff
@@ -95,7 +95,7 @@ def __init__(self, config=None, **kw):
         with_default_config = self.default_config
         if config:
             with_default_config.merge(config)
-        
```

Comment:
Please configure your editor not to strip trailing whitespace, it produces unnecessary diffs.